app/services/foreman_ansible/fact_parser.rb

```diff
@@ -64,6 +64,10 @@ def get_facts_for_interface(interface)
 
     def ipmi_interface; end
 
+    def boot_time
+      Time.zone.now.to_i - facts['ansible_uptime_seconds'].to_i
```

Comment:
Are we ok with our idea of boot time being off by a couple of seconds (or minutes, depending on the how long the job run)?

Reply:
That is the case for every facts parser, it will differ by the time facter was ran and we parsed it on satellite side. It was the case for uptime as well which even got older faster.